lib/taurus/core/tango/tangovalidator.py

```diff
@@ -120,10 +123,10 @@ def getNames(self, fullname, factory=None, queryAuth=True):
 
         if default_authority is None:
             import PyTango
-            host, port = PyTango.ApiUtil.get_env_var('TANGO_HOST').split(":")
-            # Get the fully qualified domain name
-            host = socket.getfqdn(host)
-            default_authority = "//{0}:{1}".format(host, port)
+            tango_host = PyTango.ApiUtil.get_env_var('TANGO_HOST')
+            v = TangoAuthorityNameValidator()
+            g = v.getUriGroups('tango://{}'.format(tango_host))
```

Comment:
Is this taking care of the fqdn?

Reply:
Yes, it is. `TangoAuthorityNameValidator.getUriGroups` does the conversion